leapp/utils/report.py

```diff
@@ -18,3 +18,19 @@ def fetch_upgrade_report_messages(context_id):
         messages.append(report)
 
     return messages
+
+
+def generate_report_file(messages_to_report, path):
+    if path.endswith(".txt"):
+        with open(path, 'w') as f:
+            for message in messages_to_report:
+                f.write('Severity: {}\n'.format(message['severity']))
+                f.write('Title: {}\n'.format(message['title']))
+                f.write('Summary: {}\n'.format(message['summary']))
+                if message.get('detail', {}).get('remediations'):
+                    f.write('Remediation: {}\n'.format(','.join(["{}: {}".format(r['type'], r['context'])
```

Comment:
Ahh, so when `type == command` we get this:
```
[u'yum', u'remove', u'leapp leapp-deps leapp-pes-data leapp-repository leapp-repository-deps leapp-repository-sos-plugin python2-leapp']
```
So I guess that we want something like `' '.join(...)` -- I think that we want to move this to a separate function like `_write_remediations()` for readability

Reply:
I'd rather have a class method like repr() for each Remediation type than one function, but you are right